org.free_astro.siril.json

```
@@ -30,8 +30,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-3.49.tar.gz",
-                    "sha256": "5b65a20d5c53494ec8f638267fca4a629836b7ac8dd0ef0266834eab270ed4b3"
+                    "url": "http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-4.0.0.tar.gz",
+                    "sha256": "88735b69bb017ee92823b9e7ee14efeae2e57fb6c00503ed3aa2820e850a7b5e"
```

```suggestion
                    "url": "https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfit-4.0.0.zip",
                    "sha256": "b70423f831dc28919c56ff37baebb50b1eaca665ca8ca34094cb58b60c482386"
```

this is where it is located.

---

nevermind